guides/common/modules/proc_installing-fapolicyd-on-server.adoc

```diff
@@ -0,0 +1,52 @@
+[id="installing-fapolicyd-on-server_{context}"]
+ifeval::["{context}" == "{project-context}"]
+= Installing fapolicyd on {ProjectServer}
+endif::[]
+ifeval::["{context}" == "{smart-proxy-context}"]
+= Installing fapolicyd on {SmartProxyServer}
+endif::[]
+
+ifeval::["{context}" == "{project-context}"]
+You can install `fapolicyd` along with {ProjectServer} or can be installed on an existing {ProjectServer}.
+If you are installing fapolicyd along with the new {ProjectServer}, the installation process will detect the fapolicyd in your {EL} host and deploy the {Project} rules automatically.
+endif::[]
+ifeval::["{context}" == "{smart-proxy-context}"]
+You can install `fapolicyd` along with {SmartProxyServer} or can be installed on an existing {SmartProxyServer}.
+If you are installing fapolicyd along with the new {SmartProxyServer}, the installation process will detect the fapolicyd in your {EL} host and deploy the {SmartProxy} rules automatically.
+endif::[]
+
+.Prerequisite
+* Ensure your host has access to the base repositories of {EL}.
```

Comment:
Do you mean "BaseOS" repository or default repositories "BaseOS + AppStream"?

Reply:
`BaseOS` is must. No mentioning of `AppStream` till now. Renamed `base` to `BaseOS` :+1: